GitTfs.Vs2017/TfsHelper.Vs2017.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.TeamFoundation.Build.Client;
+using Microsoft.TeamFoundation.Client;
+using Microsoft.TeamFoundation.Server;
+using Microsoft.TeamFoundation.VersionControl.Client;
+using Microsoft.VisualStudio.Setup.Configuration;
+using Sep.Git.Tfs.Core.TfsInterop;
+using Sep.Git.Tfs.VsCommon;
+using StructureMap;
+
+namespace Sep.Git.Tfs.Vs2017
+{
+    public class TfsHelper : TfsHelperBase
+    {
+        private const int REGDB_E_CLASSNOTREG = unchecked((int)0x80040154);
+
+        private const string tfsExtensionsFolder =
+            @"Common7\IDE\CommonExtensions\Microsoft\TeamFoundation\Team Explorer";
+
+        protected string TfsVersionString { get { return "15.0"; } }
+
+        public TfsHelper(TfsApiBridge bridge, IContainer container)
+            : base(bridge, container)
+        { }
+
+        protected override bool HasWorkItems(Changeset changeset)
+        {
+            return Retry.Do(() => changeset.AssociatedWorkItems.Length > 0);
+        }
```

Comment:
You know, I was thinking about this, and I think I can simplify the Paket Dependencies. Because @pmiossec updated all the projects to compile against .NET 4.6.2, there's no reason for the VS2015 plugin to be "locked down" to using the v14 Team Foundation client libraries. That plugin could also make use of the v15 libraries. This would eliminate the need for the "paket dependency groups" I created. But, this would also mean that the 2015 plugin also would not be able to make use of the TfsPluginYou know, I was thinking about this, and I think I can simplify the Paket Dependencies. Because @pmiossec updated all the projects to compile against .NET 4.6.2, there's no reason for the VS2015 plugin to be "locked down" to using the v14 Team Foundation client libraries. That plugin could also make use of the v15 libraries. This would eliminate the need for the "paket dependency groups" I created. But, this would also mean that the 2015 plugin also would not be able to make use of the `TfsHelperVs2012Base` class because of the breaking changes in method signatures introduced in v15 of the Team Foundation client libraries.